org.ghidra_sre.Ghidra.json

```diff
@@ -48,9 +48,10 @@
             "name": "ghidra",
             "buildsystem": "simple",
             "build-commands": [
-                "icotool -x support/ghidra.ico -i 8",
-                "mkdir /app/ghidra",
-                "cp -r * /app/ghidra",
+                "source /usr/lib/sdk/openjdk11/enable.sh && gradle buildGhidra",
```

Comment:
You'll probably want to reorder the commits (`git rebase -i origin/master`) so this commit still builds. The dependencies haven't been added to the manifest, so it's not going to work.

Reply:
Done